docs/setup-guide.md

```diff
@@ -0,0 +1,371 @@
+
+```shell
+[root@keylime-fedora ~]# ls /root/payload
+
+action_list autorun.sh id_rsa id_rsa.pub local_action_remove_ssh.py
+```
+
+Now execute the tenant command including the CA:
+
+```shell
+[root@keylime-fedora ~]# keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u d432fbb3-d2f1-4a97-9ef7-75bd81c00000 --cert /root/myca --allowlist /root/allowlist.txt --exclude /root/excludes.txt -f—-include /root/payload -c add
+```
+
+Run keylime_ca
+
+```shell
+[root@keylime-fedora ~]# keylime_ca -c listen -d /root/myca
+```
+
+## Troubleshooting
+
+Here are some solutions to possible errors bringing the Vagrant box up encountered on a clean Fedora 36 install.
+
+```shell
+==> keylime-fedora1: Creating shared folders metadata…
+==> keylime-fedora1: Starting domain. /usr/share/gems/gems/fog-libvirt-0.8.0/lib/fog/libvirt/requests/compute/vm_action.rb:7:in `create': Call to virDomainCreateWithFlags failed: internal error: /usr/libexec/qemu-bridge-helper --use-vnet --br=virbr0 --fd=29: failed to communicate with bridge helper: Transport endpoint is not connected (Libvirt::Error) stderr=failed to get mtu of bridge `virbr0': No such device
+```
+
+This error might have come up because of an incomplete installation of libvirt. Check your firewall zones with this command:
+
+```shell
+[localhost ~]$ firewall-cmd --get-active-zones
+FedoraWorkstation
+  interfaces: wlp0s20f3
+docker
+  interfaces: docker0
+libvirt
+  interfaces: virbr0
+```
+
+You should see a zone for libvirt present (like above); if not, that’s likely what’s causing the error. A possible fix is enabling/starting or just restarting `libvirtd` (though make sure it’s enabled if restarting):
+
+```shell
+[localhost ~]$ sudo systemctl enable libvirtd
+[localhost ~]$ sudo systemctl start libvirtd
+[localhost ~]$ sudo systemctl restart libvirtd
+```
+
+Check firewall zones again. If the libvirt zone is present, the issue should be fixed.
+
+```shell
+/usr/share/vagrant/gems/gems/vagrant-libvirt-0.7.0/lib/vagrant-libvirt/driver.rb:158:in 'list_all_networks': Call to virConnectListAllNetworks failed: Failed to connect socket to '/var/run/libvirt/virtnetworkd-sock-ro': No such file or directory (Libvirt::RetrieveError)
+```
+
+This error could be caused by libvirt networking being offline. Check status of `virtnetworkd`:
+
+```shell
+[localhost ~]$ sudo systemctl status virtnetworkd
+```
+
+If disabled and inactive, try enabling and starting it, then trying again.
+
+```shell
+[localhost ~]$ sudo systemctl enable virtnetworkd
+[localhost ~]$ sudo systemctl start virtnetworkd
+```
+
+## Using multiple Vagrant instances together
+
+The Keylime vagrantfile is already set up to easily provision multiple instances of the same VM if desired, and that’s what will be used here.
+
+Before doing anything, open up the Vagrantfile and uncomment the lines with forwarded ports from here and below: https://github.com/keylime/keylime-vagrant-ansible-tpm-emulator/blob/master/Vagrantfile#L65
```

Comment:
I'm worried this might be too fragile. If that file gets updated in anyway to create or remove a line this documentation is invalid. Maybe reference the text of the comment instead? That's less likely to accidentally change.

Reply:
Done in my latest commit.